master/buildbot/www/oauth2.py

```diff
@@ -51,7 +54,7 @@ def renderLogin(self, request):
                 details.update(infos)
             request.getSession().user_info = details
             state = request.args.get("state", [""])[0]
-            print repr(state)
+            log.msg(repr(state))
```

Comment:
it might be, however since it's in the code, i'd like to make it Python 3 compatible.